src/main/java/com/vonage/quickstart/voice/ModifyCall.java

```diff
@@ -20,34 +20,32 @@
  * THE SOFTWARE.
  */
 
-package com.nexmo.quickstart.voice;
+package com.vonage.quickstart.voice;
 
-import com.nexmo.client.NexmoClient;
-import com.nexmo.client.voice.Call;
-import com.nexmo.client.voice.CallEvent;
-import com.nexmo.client.voice.ModifyCallAction;
-
-import static com.nexmo.quickstart.Util.configureLogging;
-import static com.nexmo.quickstart.Util.envVar;
+import com.vonage.client.VonageClient;
+import com.vonage.client.voice.Call;
+import com.vonage.client.voice.CallEvent;
+import com.vonage.client.voice.ModifyCallAction;
+import com.vonage.quickstart.Util;
 
 public class ModifyCall {
     public static void main(String[] args) throws Exception {
-        configureLogging();
+        Util.configureLogging();
 
-        String NEXMO_NUMBER = envVar("NEXMO_NUMBER");
-        String TO_NUMBER = envVar("TO_NUMBER");
+        String VONAGE_NUMBER = Util.envVar("VONAGE_NUMBER");
+        String TO_NUMBER = Util.envVar("TO_NUMBER");
 
-        String NEXMO_APPLICATION_ID = envVar("NEXMO_APPLICATION_ID");
-        String NEXMO_PRIVATE_KEY_PATH = envVar("NEXMO_PRIVATE_KEY_PATH");
+        String VONAGE_APPLICATION_ID = Util.envVar("VONAGE_APPLICATION_ID");
+        String VONAGE_PRIVATE_KEY_PATH = Util.envVar("VONAGE_PRIVATE_KEY_PATH");
 
-        NexmoClient client = NexmoClient.builder()
-                .applicationId(NEXMO_APPLICATION_ID)
-                .privateKeyPath(NEXMO_PRIVATE_KEY_PATH)
+        VonageClient client = VonageClient.builder()
+                .applicationId(VONAGE_APPLICATION_ID)
+                .privateKeyPath(VONAGE_PRIVATE_KEY_PATH)
                 .build();
 
         CallEvent call = client.getVoiceClient().createCall(new Call(
                 TO_NUMBER,
-                NEXMO_NUMBER,
+                VONAGE_NUMBER,
                 "https://gist.githubusercontent.com/ChrisGuzman/d6add5b23a8cf913dcdc5a8eabc223ef/raw/a1eb52e0ce2d3cef98bab14d27f3adcdff2af881/long_talk.json"
```

Comment:
Since we are in the migration mode - I wonder if we can find a better location for this example?

Reply:
Maybe a gist from vonage or just within the repo?

Reply:
the NCCOs were being moved to the ncco-example repo.

Reply:
I'll get something made so we can remove these.

Reply:
You can use this here now - https://nexmo-community.github.io/ncco-examples/

Reply:
yeah, I was just saying that ncco-examples repo was where I would need to move them too if they don't already exist